op-batcher/metrics/metrics.go

```diff
@@ -220,6 +228,10 @@ func (m *Metrics) RecordChannelClosed(id derive.ChannelID, numPendingBlocks int,
 	m.ChannelInputBytes.WithLabelValues(StageClosed).Set(float64(inputBytes))
 	m.ChannelOutputBytes.Set(float64(outputComprBytes))
 
+	outputBytesDelta := outputComprBytes - m.lastChannelOutputBytes
+	m.lastChannelOutputBytes = outputComprBytes
```

Comment:
`outputComprBytes` comes from `s.pendingChannel.OutputBytes()` and it should only be called once per channel so I don't think we need to subtract the previous count.

Reply:
yup. I'll fix that.
I'm also wondering if we can just use the existing gauge to compute the output data rate. I'm playing with promql's `deriv` function to see if could work instead.

Reply:
IMO it's better to have a counter & then compute the diff/rate to figure out what the guage would have read. I think we should add this counter.